docs/modules/ROOT/pages/def.adoc

```diff
@@ -0,0 +1,46 @@
+= clc def
+
+Define a variable with the given name and value.
+
+NOTE: This command is not available in non-interactive mode.
+
+Usage:
+
+[source,bash]
+----
+clc def [name] [value]
+----
+
+Examples:
+
+- Setting the shell prompt:
+
+[source,bash]
+----
+\def prompt "{{config_name}}@SQL> "
+----
+
+- Configuring the banner:
+
+[source,bash]
+----
+\def banner "Hazelcast CLC {{.version}} (c) 2023 Hazelcast Inc. {{nl}}
+Log Path : {{.log_path}}"
+----
+
+In these examples, `config_name` action, and `version` and `log_path` placeholders are used.
```

Comment:
the `config_name` action, and the `version`....

Reply:
https://github.com/hazelcast/clc-docs/pull/5/commits/156434bc9a0369bc6690f0c7071ef6942ee01478